doc/security/cryptolib/contributing.md

```diff
@@ -0,0 +1,131 @@
+# Contribution Guidelines for the OT Cryptolib
```

Comment:
nit: Can you add a section recommending to update the documentation whenever API visible changes are made?